code/controllers/subsystem/air.dm

```diff
@@ -122,12 +122,12 @@ var/datum/subsystem/air/SSair
 
 
 /datum/subsystem/air/proc/process_high_pressure_delta()
-	for(var/turf/T in high_pressure_delta)
+	for(var/O in high_pressure_delta)
+		var/turf/open/T = O
```

Comment:
here you're assuming that all elements in the high_pressure_delta list is a /turf/open, but that list is defined above as var/list/turf/high_pressure_delta, not var/list/turf/open/high_pressure_delta. Which is right?